modules/next/composer.json

```diff
@@ -12,11 +12,14 @@
   ],
   "require": {
     "drupal/decoupled_router": "^2.0",
+    "drupal/graphql": "^4.8",
+    "drupal/graphql_compose": "^2.2",
+    "drupal/jwt": "^2 || ^3",
```

Comment:
This seems unrelated to the D11 upgrade.

Reply:
This has been resolved.